randr.go

```diff
@@ -144,6 +170,41 @@ func (m Modes) String() string {
 	return strings.Join(str, " ")
 }
 
+// Generates the monitor id from the edid
+func GenerateMonitorId(edid string) (string, error) {
```

Comment:
I think it'd be of great value to add a table driven test just for this function so that we see when it breaks in the future.

Reply:
Added a test in `randr_test.go`, where also the failures are tested for too short EDIDs and a wrong flag.

Reply:
Awesome.